src/Compilers/Test/Core/SourceGeneration/TestGenerators.cs

```diff
@@ -199,3 +199,18 @@ public IncrementalAndSourceCallbackGenerator(Action<GeneratorInitializationConte
         public void Initialize(IncrementalGeneratorInitializationContext context) => _onInit(context);
     }
 }
+
+namespace Microsoft.NET.Sdk.Razor.SourceGenerators
+{
+    internal sealed class RazorSourceGenerator : ISourceGenerator
```

Comment:
are the names of these types important?  if so, consider mentioning that, as well as what purpose thsi particular generator services.